frontend/src/components/Header.tsx

```diff
@@ -161,21 +159,23 @@ const Header = ({ logoPath, locale }: Props) => {
         }}
       />
       <GovBanner language={language} />
-      <USWDSHeader basic={true}>
+      <USWDSHeader
+        basic={true}
+        className="desktop:position-sticky top-0 desktop:z-500 bg-white border-bottom-2px border-primary-vivid"
+      >
         <div className="usa-nav-container">
-          <div className="usa-navbar">
-            <Title className="desktop:margin-top-2">
+          <div className="usa-navbar border-bottom-0">
+            <Title className="margin-y-2">
               <div className="display-flex flex-align-center">
-                {logoPath && (
-                  <span className="margin-right-1">
+                <Link href="/" className="display-block">
+                  {logoPath && (
                     <img
-                      className="width-3 desktop:width-5 text-bottom margin-right-05"
                       src={assetPath(logoPath)}
-                      alt="Site logo"
+                      alt={t("title")}
+                      className="display-block height-4  desktop:height-auto"
```

Comment:
The `height` sets the logo size on small screens. On larger screens logo size is set by width, so removing the `height` on desktop.